docs/introvideos/basics.md

```diff
@@ -6,20 +6,22 @@ MetaSocialImage: images/opengraph/introvideos-social.png
 ---
 # Getting started with Visual Studio Code
 
-In this tutorial, we walk you through setting up Visual Studio Code and give an overview of the basic features.
+In this tutorial, we walk you through using Visual Studio Code powered with AI features.
 
-<iframe src="https://www.youtube-nocookie.com/embed/B-s71n0dHUk?autoplay=true" width="640" height="320" allowFullScreen="true" frameBorder="0" title="Getting Started with Visual Studio Code"></iframe>
+<iframe src="https://www.youtube.com/embed/5NxGqnTazR8?si=UaGTeedyAF_qv8tZ?autoplay=true" width="640" height="320" allowFullScreen="true" frameBorder="0" title="Getting Started with Visual Studio Code"></iframe>
```

Comment:
@cwebster-99 embedded YT links need to use the`www.youtube-nocookie.com` domain for user privacy reasons.